### PR TITLE
Option to force scheme and host at application level (fixes #159)

### LINKED ISSUE #159: Make service url + scheme configurable
We should have a way to define the service URL / scheme so it's set explicitely.

ref. https://github.com/mozilla-services/cliquet/pull/151

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Comment by Natim** [comment]
I like this :)

**Comment by leplatrem** [comment]
@Natim @ametaireau r?

File: docs/configuration.rst [hunk 1]
```diff
@@ -50,6 +50,29 @@ Deployment
...
 
+
+Scheme, host and port
+:::::::::::::::::::::
+
+By default *cliquet* does not enforce requests scheme, host and port. It relies
+on WSGI specification and the related stack configuration. Tuning this becomes
+necessary when the application runs behind proxies or load balancers.
+
+Most implementations, like *uwsgi, provide configuration variables to adjust it
```
Comment (ametaireau) [inline docs/configuration.rst]: misses a * ?

**Comment by ametaireau** [comment]
r+, great.

**Pull request merged**